activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+*   Introduce ApplicationRecord, an Active Record layer super type.
+
+    An `ApplicationRecord` let's engines have models, isolated from the main
+    application. Plugin authors can use it to distribute extensions as modules
+    to be included into `ApplicationRecord`, instead of monkey patches. It can
+    also serve as a place for applications to customize the default
+    `ActiveRecord::Base` model behaviour.
```

Comment:
This is important, but not so relevant to app developers reading the change log. Good discussion for a guide on plugin/engine development.